storage/src/qmdb/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! A collection of authenticated databases (ADB).
+//! A collection of authenticated databases.
```

Comment:
nit: `A collection of authenticated databases inspired by QMDB (Quick Merkle Database).`

Reply:
Done